go.mod

```diff
@@ -13,22 +13,30 @@ require (
 	github.com/iancoleman/strcase v0.1.3
 	github.com/imdario/mergo v0.3.12
 	github.com/jonboulle/clockwork v0.1.0
+	github.com/json-iterator/go v1.1.11 // indirect
```

Comment:
Can't this be done by using `replace` in the CLI go.mod ?

Reply:
https://golang.org/ref/mod#go-mod-file-replace

Reply:
This comment is for all downgrades intended .. don'T want to repeat it several times :)